src/App.tsx

```diff
@@ -46,6 +49,12 @@ const CustomListView = ({ program, dispatch }) => {
 export const cardEditorStrKey = "cardEditorStrKey";
 const updateViewReducer = (state, action) => {
   switch (action.type) {
+    case "set card editor": {
+      return {
+        ...state,
+        cardEditor: action.cardEditor,
+      };
+    }
```

Comment:
new dispatch action to allow CardEditor to change in response to sibling CommandEditor changes (namely, load decks)